codalab/lib/beam/mockblobstoragefilesystem.py

```diff
@@ -34,7 +34,10 @@ def _azfs_to_local(self, path):
         if not path.startswith("azfs://"):
             return path
         path = MockBlobStorageFileSystem.AZFS_MOCK_LOCATION + path[len("azfs://") :]
-        os.makedirs(os.path.dirname(path), exist_ok=True)
+        try:
```

Comment:
This is only used during tests. This change is needed because `os.makedirs(os.path.dirname(path), exist_ok=True)` fails when the path has a trailing slash.